src/components/Order/index.tsx

```diff
@@ -151,6 +155,15 @@ export class Order extends React.Component<OrderComponentProps, State> {
         amountBuy: '',
     };
 
+
+    public componentDidMount() {
+        const { defaultTabIndex } = this.props;
+
+        if (defaultTabIndex !== undefined) {
```

Comment:
should we check for undefined or it can be null as well?

Reply:
it can be only number or undefined)